.github/workflows/ci.yml

```diff
@@ -0,0 +1,86 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    # NOTE: Don't add other branches
+    branches:
+      - release
+
+jobs:
+  test:
+    name: Run Tests
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: rlespinasse/github-slug-action@v3.x
+
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+      - uses: actions/cache@v1
+        id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
+      - name: Run yarn install
+        run: yarn install
+
+      - name: Run yarn build (also test)
+        run: env > .env && NODE_OPTIONS=--max_old_space_size=4096 yarn build:unsafe
```

Comment:
@tnagorra @AdityaKhatri. Will need to update this.
![image](https://user-images.githubusercontent.com/7059255/121993519-5434df00-cdc3-11eb-9bc8-f50872cf21be.png)

Reply:
We don't have "yarn build:unsafe", it's just "yarn build" now @AdityaKhatri if you could fix it